lib/gitlab_merge_request_hook.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'gitlab_init'
+
+class GitlabMergeRequestHook
+  attr_reader :config, :repo_path, :changes
+
+  def initialize(repo_path, config, changes)
+    @branch    = branch_name(changes)
+    @config    = config
+    @repo_path = repo_path
+  end
+
+  def exec
+    return if @branch.nil? || @branch == "master"
+    puts
+    puts "To open a merge request for #{@branch}, enter in:"
+    puts "\t#{repo_url}"
+    puts
+  end
+
+  private
+
+  def branch_name(changes)
+    changes
+      .encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '')
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.